tests/ci/cdk/cdk/aws_lc_github_actions_stack.py

```diff
@@ -60,7 +70,27 @@ def __init__(
                             "ecr:BatchCheckLayerAvailability",
                             "ecr:GetDownloadUrlForLayer",
                         ],
-                        resources=[x.repository_arn for x in ecr_repos],
+                        resources=[x for x in itertools.chain([
+                            x.repository_arn for x in ecr_repos
+                        ], [x.repository_arn for x in pull_through_caches])],
+                    ),
+                    iam.PolicyStatement(
+                        effect=iam.Effect.ALLOW,
+                        actions=[
+                            "ecr:CompleteLayerUpload",
+                            "ecr:InitiateLayerUpload",
+                            "ecr:PutImage",
+                            "ecr:UploadLayerPart",
```

Comment:
prior policy allowed

```
                            "ecr:BatchGetImage",
                            "ecr:BatchCheckLayerAvailability",
                            "ecr:CompleteLayerUpload",
                            "ecr:GetDownloadUrlForLayer",
                            "ecr:InitiateLayerUpload",
                            "ecr:PutImage",
                            "ecr:UploadLayerPart",
```

you've confirmed that those extra permissions are no longer necessary?